packages/react-divider/src/components/Divider/renderDivider.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utilities';
+import { DividerState } from './Divider.types';
+import { dividerShorthandProps } from './useDivider';
+
+/**
+ * Function that renders the final JSX of the component
+ */
+export const renderDivider = (state: DividerState) => {
+  const { slots, slotProps } = getSlots(state, dividerShorthandProps);
+  const { children, labelledById } = state;
+
+  return (
+    <slots.root {...slotProps.root} role="separator" aria-labelledby={labelledById}>
+      {children && <div id={labelledById}>{children}</div>}
```

Comment:
In this case you should refactor to create a new slot so that it's configurable by consumers

```suggestion
      {children && <slots.wrapper {...slotProps.wrapper} />}
```

Reply:
I'm not an expert on `aria-labelledby` but is it really needed here, if the labeling content is a direct child of the divider?

If it is needed, can the ID be added to the root div, and have aria-labelledby refer to itself, rather than adding an extra wrapper slot?

Reply:
Tagging @smhigley for the aria-labelledby question.

Reply:
updated to slot